app/models/message.rb

```diff
@@ -31,6 +31,25 @@ def create_from_queue(body)
     self
   end
 
+  def set_status!
+    successes = 0
+
+    transmissions = Transmission.where(:message_uuid => self.uuid)
+    transmissions.each do |t|
```

Comment:
Can we use `transmission` as the variable not `t` ? Just a ruby thing. We should get your setup with rubocop so that it tells you stuff like this.